sdk/src/main/scala/io/horizen/account/state/AccountState.scala

```diff
@@ -377,9 +385,11 @@ class AccountState(
 
   override def hasCeased: Boolean = stateMetadataStorage.hasCeased
 
-  override def getFeePaymentsInfo(withdrawalEpoch: Int, blockToAppendFeeInfo: Option[AccountBlockFeeInfo] = None): Seq[AccountPayment] = {
+  override def getFeePaymentsInfo(withdrawalEpoch: Int, consensusEpochNumber: ConsensusEpochNumber, blockToAppendFeeInfo: Option[AccountBlockFeeInfo] = None): Seq[AccountPayment] = {
```

Comment:
This method is called in AbstractSidechainNodeViewHolder after having applied the block for saving the fees inside the History. The problem is that the rewards from the Forger_Poll have already been canceled, so at this point they are always 0. Maybe these fees need to be saved also in the metadatastorage